src/python/pants/engine/native.py

```diff
@@ -869,7 +869,8 @@ def setup_pantsd_logger(self, log_file_path, level):
     def setup_stderr_logger(self, level):
         return self.lib.setup_stderr_logger(level)
 
-    def write_log(self, msg, level, target):
+    def write_log(self, *, msg: str, level: int, target: str):
```

Comment:
Nit, I'd recommend:

```suggestion
    def write_log(self, msg: str, *, level: int, target: str):
```

`msg` is pretty intuitive. Kwargs shine when you can't easily anticipate what the value corresponds to, e.g. in `write_log("hello!", 2)`, the 2 is not at all obvious.